include/workspaces.h

```diff
@@ -2,6 +2,9 @@
 #ifndef LABWC_WORKSPACES_H
 #define LABWC_WORKSPACES_H
 
+#include <stdbool.h>
+#include <wayland-util.h>
+
```

Comment:
Nicely spotted the missing `<wayland-util.h>` include.

`struct view` and `struct wl_list` forward declares can be removed.
`struct wlr_scene_tree` forward declare is missing.

Reply:
Fixed